mto_route_product_variant/models/product_product.py

```diff
@@ -0,0 +1,63 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, models, fields
+
+IS_MTO_HELP = """
+    Check or Uncheck this field if you want this variant to have another
+    mto configration than its template.
```

Comment:
The help is confusing.

To me, is_mto means the product has MTO route activated, whereas "another configuration than its template" does not mean that. I understand it as if the template is MTO, the variant is not, or if the template is not MTO, the variant is.